morpheus/stages/inference/triton_inference_stage.py

```diff
@@ -756,7 +756,8 @@ def __init__(self,
 
     def supports_cpp_node(self) -> bool:
         # Get the value from the worker class
-        return TritonInferenceWorker.supports_cpp_node()
+        # The C++ impl of this stage does not support the use_shared_memory options
+        return (TritonInferenceWorker.supports_cpp_node() and not self._use_shared_memory)
```

Comment:
Generate a warning message to show that the C++ impl isnt being used. Lets make a bug to fix this in the future.